### ISSUE TITLE
Loose zoom level on the map view when widget page refreshes

### ISSUE THREAD

**Issue by arthurlogilab** [shinken-monitoring/mod-webui#412]
I have a worldmap widget that I then move around and change the zoom. That setting is not saved and is lost when the page refreshes.

**Comment by mohierf** [comment]
You are right. The map widget should be improved ... to be included in a next release.

**Comment by mohierf** [comment]
Implementing this feature requires a quite huge widget refresh refactoring ... postponed!
